documentation/test/warn_only_tests.md

```diff
@@ -1,12 +1,12 @@
 # Warn only `test`
-By default, a failing `test` will set `hasValidationErrors` to `true`. Sometimes you would want to set a warn-only validation test (password strength, for example). In this case, you would add the `'warn'` flag to your test function.
+By default, a failing `test` will set `hasValidationErrors` to `true`. Sometimes you would want to set a warn-only validation test (password strength, for example). In this case, you would add the `'warn'` flag to your test function as the last argument.
 This will leave `hasValidationErrors` unchanged (other tests may have set it to `true`), and update `hasValidationWarnings` to `true`. It will also bump up `warnCount`.
 
 If no flag is added, your test function will default to `fail`.
 
 ```js
 Passable('WarnAndPass', (test) => {
-    test('WarnMe', 'Should warn and not fail', 'warn', () => false);
+    test('WarnMe', 'Should warn and not fail', () => false, 'warn');
```

Comment:
(question not linked to this PR) Passable is a class, why you are not doing `new Passable` ? I understand that when the consumer write `Passable('dataName', (test) => {...})` the `test` function is define [here](https://github.com/fiverr/passable/blob/90b45126270af0b0e69dbe16a8c18102508b8ae2/src/Passable.js#L34) right? But I can't find where this: `Passable('dataName', (test) => {...})` magic happen?